cypress/integration/group1/checkerWorkflowFromWorkflow.ts

```diff
@@ -55,14 +55,13 @@ describe('Checker workflow test from my-workflows', () => {
       cy.fixture('refreshedChecker').then((json) => {
         cy.route({
           method: 'GET',
-          url: '/workflows/*/refresh',
+          url: '/api/workflows/*/refresh',
```

Comment:
`/api` is supposed to be part of the API URL.  Test changes should not be necessary

Reply:
I like it because it more closely matches the UI invocations in staging/production. But it was unfortunate I had to change all the test files.

Reply:
so for production, if we end up changing the web service to dockstore-webservice.org with no /api, we'll have to get rid of all these again :cry:

Reply:
Good point.